### PR TITLE
getting rid of 'pipelien'

### PR DESCRIPTION
**Pull request by ChaosInTheCRD**
Signed-off-by: ChaosInTheCRD <thomas.meadows@jetstack.io>

# Changes

Just found a typo in the codebase, getting rid of it.

# Submitter Checklist

As the author of this PR, please check off the items in this checklist:

- [x] Has [Docs](https://github.com/tektoncd/community/blob/main/standards.md#docs) included if any changes are user facing
- [x] Has [Tests](https://github.com/tektoncd/community/blob/main/standards.md#tests) included if any functionality added or changed
- [x] Follows the [commit message standard](https://github.com/tektoncd/community/blob/main/standards.md#commits)
- [x] Meets the [Tekton contributor standards](https://github.com/tektoncd/community/blob/main/standards.md) (including
  functionality, content, code)
- [x] Release notes block below has been updated with any user facing changes (API changes, bug fixes, changes requiring upgrade notices or deprecation warnings)
- [x] Release notes contains the string "action required" if the change requires additional action from users switching to the new release

# Release Notes

``` release-note
NONE
```

### CODE REVIEW

File: vendor/github.com/tektoncd/pipeline/pkg/apis/config/metrics.go [hunk 1]
```diff
@@ -50,7 +50,7 @@ const (
 	// when it isn't specified in configmap
 	DefaultPipelinerunLevel = PipelinerunLevelAtPipeline
 	// PipelinerunLevelAtPipelinerun specify that aggregation will be done at
-	// pipelienrun level
```
Comment (wlynch) [inline vendor/github.com/tektoncd/pipeline/pkg/apis/config/metrics.go:53]: This file is imported from https://github.com/tektoncd/pipeline - we'll need to make the change upstream and pull it in.

For this PR, you can revert this file.

**Review by wlynch — CHANGES_REQUESTED** [review]

**Review by priyawadhwa — APPROVED** [review]

**Review by Yongxuanzhang — COMMENTED** [review]
/lgtm
Thanks for fixing this! I agree with Billy's comments!

File: vendor/github.com/tektoncd/pipeline/pkg/apis/config/metrics.go (hunk 1 above)
Comment (chuangw6) [reply vendor/github.com/tektoncd/pipeline/pkg/apis/config/metrics.go:53]: Fixed in pipeline https://github.com/tektoncd/pipeline/pull/5382. Using same title ;)

Comment (lcarva) [reply vendor/github.com/tektoncd/pipeline/pkg/apis/config/metrics.go:53]: I think we still want to revert the file, right? Anything under vendor will be updated on the next module update.

Comment (ChaosInTheCRD) [reply vendor/github.com/tektoncd/pipeline/pkg/apis/config/metrics.go:53]: sure thing 🙂

Comment (ChaosInTheCRD) [reply vendor/github.com/tektoncd/pipeline/pkg/apis/config/metrics.go]: updated - would be good if I could get a re-review

**Pull request merged**